projects/storefrontapp-e2e-cypress/cypress/e2e/vendor/cpq/download-proposal-document.e2e.cy.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
```

Comment:
@suprishi, could you, please, rename the file to **cpq-**download-proposal-document.e2e.cy.ts.
Thank you!